geoportailv3/static/js/maincontroller.js

```diff
@@ -473,8 +473,10 @@ app.MainController = function(
         return this.ngeoNetworkStatus.offline;
       }, () => {
       if (this.ngeoNetworkStatus.offline) {
-        this.closeSidebar();
-        this.layersOpen = true;
+        if (this.sidebarOpen() && !this['layersOpen'] && !this['mymapsOpen']) {
+          this.closeSidebar();
+          this['layersOpen'] = true;
```

Comment:
Do we really favour `this['layersOpen']` over `this.layersOpen` ?

Reply:
That always looks like this below in the code, so I though there is a reason for. I didn't try with the dot notation

Reply:
Ok i guess it's the way to do the `@export` thing.
Please merge.

Reply:
That's not nice, but that's coherent... :-/